libs/features/georoutingv2/georoutingv2.js

```diff
@@ -166,6 +166,15 @@ function openPicker(button, locales, country, event, dir) {
     list.appendChild(li);
   });
   button.parentNode.insertBefore(list, button.nextSibling);
+  const buttonRect = button.getBoundingClientRect();
+  const windowHeight = window.innerHeight;
```

Comment:
I wouldn't even set this variable. You can just directly do 
`const spaceBelowButton = window.innerHeight - buttonRect.bottom;`